stencil.config.dist.ts

```diff
@@ -6,7 +6,7 @@ export const config: Config = {
     outputTargets: [
         {
             type: 'dist',
-            copy: [{ src: 'style/' }],
+            copy: [{ src: 'style/' }, { src: 'style/mixins.scss', dest: '../../mixins.scss' }],
```

Comment:
Since the `dist` folder already has folders like `cjs` ("common JS", aka node's kind of javascript), `esm`, and `types`, it feels reasonable to put a folder called `scss` there too, and put any exported scss files in that.

Reply:
This looks good to me of course. The point was not copy-pasting manually. If we can get the same file automatically from Elements, that'd be awesome. 

Do you want me to accept the suggestion and change all the import links?

The only thing is how to make sure that people don't add new mixins into this file? Is it an obvious thing, or does it need some comments where the file is imported?

Reply:
(So I'm doing that 😄. Will push in a few minutes.)

Reply:
> Do you want me to accept the suggestion and change all the import links?

Sorry, I didn't see that you had written this. You don't have to do anything, I've already updated all the imports in both repos.

> The only thing is how to make sure that people don't add new mixins into this file? Is it an obvious thing, or does it need some comments where the file is imported?

Ah, good point. Adding new mixins isn't a problem in itself, but they can't import anything from any file that isn't also exported, since the compilation of the file won't happen when _this_ repo is built, but when the _consumer_ repo is built.

So, yeah, I guess a comment about that might be good to have. I'll add something! 👍

Reply:
⚡